testdata/struct_tag.go

```diff
@@ -6,39 +6,41 @@ type decodeAndValidateRequest struct {
 	// BEAWRE : the flag of URLParam should match the const string URLParam
 	URLParam    string `json:"-" path:"url_param" validate:"numeric"`
 	Text        string `json:"text" validate:"max=10"`
-	DefaultInt  int    `json:"defaultInt" default:"10.0"` // MATCH /field's type and default value's type mismatch/
+	DefaultInt  int    `json:"defaultInt" default:"10.0"` // MATCH /type mismatch between field type and default value type in default tag/
 	DefaultInt2 int    `json:"defaultInt2" default:"10"`
-	// MATCH:12 /unknown option 'inline' in JSON tag/
-	DefaultInt3      int             `json:"defaultInt2,inline" default:"11"` // MATCH /duplicate tag name: 'defaultInt2'/
+	// MATCH:12 /unknown option "inline" in json tag/
+	DefaultInt3      int             `json:"defaultInt2,inline" default:"11"` // MATCH /duplicated tag name "defaultInt2" in json tag/
 	DefaultString    string          `json:"defaultString" default:"foo"`
-	DefaultBool      bool            `json:"defaultBool" default:"trues"` // MATCH /field's type and default value's type mismatch/
+	DefaultBool      bool            `json:"defaultBool" default:"trues"` // MATCH /type mismatch between field type and default value type in default tag/
 	DefaultBool2     bool            `json:"defaultBool2" default:"true"`
 	DefaultBool3     bool            `json:"defaultBool3" default:"false"`
-	DefaultFloat     float64         `json:"defaultFloat" default:"f10.0"` // MATCH /field's type and default value's type mismatch/
+	DefaultFloat     float64         `json:"defaultFloat" default:"f10.0"` // MATCH /type mismatch between field type and default value type in default tag/
 	DefaultFloat2    float64         `json:"defaultFloat2" default:"10.0"`
-	MandatoryStruct  mandatoryStruct `json:"mandatoryStruct" required:"trues"` // MATCH /required should be 'true' or 'false'/
+	MandatoryStruct  mandatoryStruct `json:"mandatoryStruct" required:"trues"` // MATCH /required should be "true" or "false" in required tag/
 	MandatoryStruct2 mandatoryStruct `json:"mandatoryStruct2" required:"true"`
 	MandatoryStruct4 mandatoryStruct `json:"mandatoryStruct4" required:"false"`
 	OptionalStruct   *optionalStruct `json:"optionalStruct,omitempty"`
 	OptionalQuery    string          `json:"-" querystring:"queryfoo"`
 	optionalQuery    string          `json:"-" querystring:"queryfoo"` // MATCH /tag on not-exported field optionalQuery/
 	// No-reg test for bug https://github.com/mgechev/revive/issues/208
 	Tiret       string `json:"-,"`
-	BadTiret    string `json:"other,"`               // MATCH /option can not be empty in JSON tag/
-	ForOmitzero string `json:"forOmitZero,omitzero"` // MATCH /unknown option 'omitzero' in JSON tag/
+	BadTiret    string `json:"other,"`               // MATCH /option can not be empty in json tag/
+	ForOmitzero string `json:"forOmitZero,omitzero"` // MATCH /unknown option "omitzero" in json tag/
```

Comment:
It's great to test this one, but while you work on refactoring everything, can't we add a message more like


```suggestion
	ForOmitzero string `json:"forOmitZero,omitzero"` // MATCH /option "omitzero" is unsupported in json tag prior Go 1.24/
```